packages/cerebral-demo/src/modules/clients/index.js

```diff
@@ -0,0 +1,53 @@
+import {input, set, state} from 'cerebral/operators'
+import addClient from './signals/addClient'
+import discardDraft from './signals/discardDraft'
+import saveDraft from './signals/saveDraft'
+import selectClient from './signals/selectClient'
+import updateDraft from './signals/updateDraft'
+
+export default {
+  state: {
+    all: {
+      'cerebral': {
+        name: 'Cerebral',
+        notes: 'Make sense of complex apps.',
+        ref: 'cerebral',
+        image: '/img/cerebral-mini.png',
+        website: 'cerebraljs.com'
+      },
+      'calvin': {
+        ref: 'calvin',
+        name: 'Calvin and Hobbes Ltd',
+        email: 'calvin@hobbes.com',
+        phone: '123456789'
+      },
+      'largo': {
+        ref: 'largo',
+        name: 'Largo Winch et al'
+      },
+      'no-client': {
+        ref: 'no-client',
+        name: 'No client'
+      }
+    },
+    $filter: '',
+    $selected: null
```

Comment:
Removed $selected, we already have ref inside `$draft.ref` but I think we need more thoughts on client editing (save, cancel buttons, click outside to ... cancel ?)